node.js

```diff
@@ -49,7 +49,42 @@ module.exports = {
     'space-before-function-paren': off,
     'keyword-spacing': error,
     'eol-last': [ error, always ],
-    'brace-style': [ error, '1tbs', { allowSingleLine: true } ],
-    'curly': [ error, all ]
+    'brace-style': [ error, '1tbs', { allowSingleLine: false } ],
+    'curly': [ error, all ],
+    'block-spacing': [ error,  always ],
+    'indent': [ error, 2, { SwitchCase: 1 } ],
+    'no-tabs': error,
+    'spaced-comment': [ error, always ],
+    'array-callback-return': warn,
+    'eqeqeq': [ error, always, { null: 'ignore' } ],
+    'no-else-return': [ warn, { allowElseIf: false } ],
+    'yoda': error,
+    'no-throw-literal': error,
+    'no-self-compare': error,
+    'no-return-assign': error,
+    'no-loop-func': error,
+    'valid-jsdoc': warn,
+    'for-direction': error,
+    'no-buffer-constructor': error,
+    'no-new-require': error,
+    'no-path-concat': error,
+    'comma-spacing': error,
+    'comma-style': error,
+    'computed-property-spacing': [ error, never ],
+    'consistent-this': [ error, 'self' ],
+    'func-call-spacing': [ error, never ],
+    'key-spacing': error,
+    'new-cap': error,
+    'no-lonely-if': error,
+    'no-mixed-operators': error,
```

Comment:
Disallow mixed operators, which can cause a huge amount of confusion.
https://eslint.org/docs/rules/no-mixed-operators
```js
// Old
var foo = a && b < 0 || c > 0 || d + 1 === 0;
var foo = a + b * c;
// New
var foo = (a && b < 0) || c > 0 || d + 1 === 0;
var foo = a + (b * c);
```
*Not auto-fixable*

Reply:
```js
a + (b * c)
``` 

seriously?

:+1:  for booleans thou

Reply:
I know, but when people are tired it's easy to forget BIDMAS, and when you're chasing down a bug you have to read through it a couple of times to double check there hasn't been a mistake. Enforcing bracketing means it's clearer, and should also try to stop developers from putting too much maths logic in a single line `a + b * c / 12 * 4 - x % 5`